libs/application/templates/party-letter/src/fields/Recommendations/CopyLink.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from 'react'
+import { FieldDescription } from '@island.is/shared/form-fields'
+import { Box, Button, Text } from '@island.is/island-ui/core'
+
+// Todo look into if this exists as util somewhere..
+const copyToClipboard = (str: string) => {
```

Comment:
There is this quite popular https://www.npmjs.com/package/copy-to-clipboard but same logic as your code

Reply:
Seems to have more compatibility will swap this out